backends/CMakeLists.txt

```diff
@@ -76,12 +76,10 @@ if(WIN32)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Qstd=c++17")
     set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -ggdb3 -DDEBUG ")
 elseif(UNIX)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}                                        \
-       -Wall -Wextra -Winit-self -Wuninitialized -Wmissing-declarations        \
-       -fdiagnostics-color=auto -O3                                            \
-    ")
+    set(SDL_FLAGS "-fstack-protector -fstack-protector-all -fpic -fPIC -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -fno-strict-overflow -fno-delete-null-pointer-checks")
```

Comment:
Can we reformat the flags like this to make them readable. Also do we not need `-O3` ?
```
    set(CMAKE_C_FLAGS "                                                        \
        ${CMAKE_C_FLAGS}                                                       \
       -Wall                                                                   \
       -Wextra                                                                 \
       -Winit-self                                                             \
       -Wuninitialized                                                         \
       -Wmissing-declarations                                                  \
       -fdiagnostics-color=auto -O3                                            \
    ")
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Qstd=c++17")
    set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -ggdb3 -DDEBUG ")
elseif(UNIX)
    set(SDL_FLAGS "                                                            \
        -fstack-protector                                                      \
        -fstack-protector-all                                                  \
        -fpic                                                                  \
        -fPIC                                                                  \
        -D_FORTIFY_SOURCE=2                                                    \
        -Wformat                                                               \
        -Wformat-security                                                      \
        -fno-strict-overflow                                                   \
        -fno-delete-null-pointer-checks                                        \
        "
    )
    set(CMAKE_C_FLAGS "                                                        \
        ${CMAKE_C_FLAGS}                                                       \
        ${SDL_FLAGS}                                                           \
        -Wall                                                                  \
        -Wextra                                                                \
        -Winit-self                                                            \
        -Wuninitialized                                                        \
        -Wmissing-declarations                                                 \
        -fdiagnostics-color=auto                                               \
        -O3                                                                    \
        "
    )
    set(CMAKE_C_FLAGS_DEBUG "                                                  \
        ${CMAKE_C_FLAGS_DEBUG}                                                 \
         ${SDL_FLAGS}                                                          \
        -Wall                                                                  \
        -Wextra                                                                \
        -Winit-self                                                            \
        -Wuninitialized                                                        \
        -Wmissing-declarations                                                 \
        -fdiagnostics-color=auto                                               \
        -O0                                                                    \
        -ggdb3                                                                 \
        -DDEBUG                                                                \
        "
    )
```

Reply:
This style of formatting makes a lot spaces in commands. You can see all calls to compilers if use `make -j4 VERBOSE=1`.

Reply:
There will be a lot of spaces.

Reply:
I would propose to use [`add_compile_options()`](https://cmake.org/cmake/help/v3.18/command/add_compile_options.html)

Reply:
Ok, Then never mind.